visualizer/ui.js

```diff
@@ -378,8 +378,21 @@ class Ui extends events.EventEmitter {
       'all-v8:cpp': 'V8 C++',
       'all-v8:regexp': 'RegExp'
     }
+
+    if (keysToLabels[key]) {
+      return keysToLabels[key]
+    }
+
+    if (key.startsWith('deps:')) {
```

Comment:
This is doing something very similar to the logic below, maybe we could replace this and that with something like 

```
    const splitKey = key.split(':')
    if (splitKey.length > 1) {
      const presetLabel = keysToLabels[splitKey[1]]
      return presetLabel || splitKey[1]
    }
```

...which will then just work if we add different types to app again.

Also fixes a mistake that slipped in to the previous code as well, `.split()`'s array is always length > 0 (and always truthy, which was a mistake before).